app/src/main/java/de/danoeh/antennapod/fragment/AllEpisodesFragment.java

```diff
@@ -397,14 +398,15 @@ public void onEventMainThread(DownloadEvent event) {
         Log.d(TAG, "onEventMainThread() called with: " + "event = [" + event + "]");
         DownloaderUpdate update = event.update;
         downloaderList = update.downloaders;
-        if(update.feedIds.length > 0) {
-            if (isUpdatingFeeds != updateRefreshMenuItemChecker.isRefreshing()) {
+        if (isUpdatingFeeds != update.feedIds.length > 0) {
```

Comment:
It is. Idea: if isUpdatingFeeds is true, feedIds.length has to be > 0 or we invalidate
if ! isUpdatingFeeds feedIds has to be zero or again, we invalidate